x-pack/plugins/synthetics/server/synthetics_service/formatters/public_formatters/browser.ts

```diff
@@ -34,7 +38,21 @@ const throttlingFormatter: Formatter = (fields) => {
 };
 
 export const browserFormatters: BrowserFormatMap = {
-  ...basicBrowserFormatters,
+  [ConfigKey.SOURCE_PROJECT_CONTENT]: null,
+  [ConfigKey.SCREENSHOTS]: null,
+  [ConfigKey.IGNORE_HTTPS_ERRORS]: null,
+  [ConfigKey.PLAYWRIGHT_OPTIONS]: null,
+  [ConfigKey.TEXT_ASSERTION]: stringToJsonFormatter,
+  [ConfigKey.PORT]: stringToJsonFormatter,
+  [ConfigKey.URLS]: stringToJsonFormatter,
+  [ConfigKey.METADATA]: objectToJsonFormatter,
+  [ConfigKey.SOURCE_INLINE]: stringToJsonFormatter,
+  [ConfigKey.SYNTHETICS_ARGS]: arrayToJsonFormatter,
+  [ConfigKey.JOURNEY_FILTERS_MATCH]: stringToJsonFormatter,
+  [ConfigKey.JOURNEY_FILTERS_TAGS]: arrayToJsonFormatter,
+  [ConfigKey.THROTTLING_CONFIG]: throttlingFormatter,
+  ...commonFormatters,
```

Comment:
Why are the common formatters spread after the browser formatters? Shouldn't the specific monitor type take precedence over common?